README.md

```diff
@@ -30,9 +30,14 @@ See [`CONTRIBUTING.md`](CONTRIBUTING.md).
   prevent commit and push actions. Please use your development environment and
   pull the changes to base.
 - **Sign your commits:**
-  - The main branch (*default branch*) has the Require signed commits (Include
-    administrators) GitHub branch protection enabled.
+  - The commit signing option has been adjusted in the repository to facilitate 
+   smoother collaboration. This modification is intended to ease the contribution 
+   process. However, it is strongly encourge that staff members continue to 
```

Comment:
I think that `encourage` should instead be `encouraged` here.

Reply:
modified.Thank you!!